examples/3d/animated_material.rs

```diff
@@ -30,14 +30,16 @@ fn setup(
     ));
 
     let cube = meshes.add(Cuboid::new(0.5, 0.5, 0.5));
+    let mut i = 0;
     for x in -1..2 {
         for z in -1..2 {
             commands.spawn(PbrBundle {
                 mesh: cube.clone(),
-                material: materials.add(Color::WHITE),
+                material: materials.add(Color::hsl((i as f32 * 2.345) * 100.0 % 360.0, 1.0, 0.5)),
                 transform: Transform::from_translation(Vec3::new(x as f32, 0.0, z as f32)),
                 ..default()
             });
+            i += 1;
```

Comment:
Not important, but you can use the `rotate_hue` and a cool property of rotating by golden ratio here for showcase:
```rs
// Not a stable f32 constant yet
const PHI: f32 = 1.6180339;

let mut c = Color::hsl(0.0, 1.0, 0.5);
for x in -1..2 {
  for z in -1..2 {
    // Pbr spawning
    c = c.rotate_hue(PHI);
  }
}
```

Reply:
As you suggested, rotating at a constant ratio makes for a smarter implementation indeed. I rotated it at the golden angle to ensure that the colors of adjacent materials diverge!